### DISCUSSION TITLE
Moving a node must not move other nodes

### DISCUSSION THREAD

**Discussion by Blue-Baloo** [freeplane/freeplane#1872] [Help]
Hello,

when I move one node, it somtimes moves the location of another node. How do I turn that off?

Thanks for helping
Blue-Baloo

**Comment by macmarrum** [comment]
Please see
- https://github.com/freeplane/freeplane/discussions/1858
- https://docs.freeplane.org/attic/new-wiki-initiative/understanding-nodes.html#free-nodes
